main_routers/live2d_router.py

```
@@ -966,3 +966,88 @@ def open_model_directory(model_name: str):
+    """获取用户导入的模型列表"""
+    try:
+        from utils.config_manager import get_config_manager
+        
+        user_models = []
+        
+        # 获取用户文档目录下的live2d模型
+        try:
+            config_mgr = get_config_manager()
+            docs_live2d_dir = str(config_mgr.live2d_dir)
+            if os.path.exists(docs_live2d_dir):
+                for root, dirs, files in os.walk(docs_live2d_dir):
```

Ruff (0.14.10)

Loop control variable `dirs` not used within loop body

Rename unused `dirs` to `_dirs`

(B007)